torch/autograd/gradcheck.py

```diff
@@ -1437,6 +1439,15 @@ def gradgradcheck(
     Returns:
         True if all differences satisfy allclose condition
     """
+    assert check_fwd_over_rev or check_rev_over_rev, \
+        "Expected at least one of check_fwd_over_rev or check_rev_over_rev to be True"
+    assert not (check_undefined_grad and not check_rev_over_rev), \
+        "Setting check_undefined_grad=True requires check_rev_over_rev to be True"
+    assert not (check_batched_grad and not check_rev_over_rev), (
+        "Setting check_batched_grad=True requires check_fwd_over_rev to be True")
+    # TODO: do we want to test this too?
+    # assert not (check_batched_forward_grad and not check_fwd_over_rev), (
```

Comment:
Is it a no-op otherwise?

Reply:
Yeah currently there is no such test (since `check_batched_forward_grad` is not passed to gradcheck). The proposal here is to add a new `check_batched_fw_over_rev` flag for `gradgradcheck`, and if `check_batched_fw_over_rev=True`, we pass `check_batched_forward_grad=True` to gradcheck.

Reply:
That sounds good!

Reply:
Saving this for a followup